webrtc/videoroom/assets/src/room_ui.ts

```diff
@@ -1,3 +1,17 @@
+import { MembraneWebRTC } from "./membraneWebRTC";
+
+interface State {
+  isLocalScreensharingActive: boolean;
+  isRemoteScreensharingActive: boolean;
+  screensharingController?: MembraneWebRTC;
```

Comment:
MembraneWebRTC should be responsible for taking tracks and sending them, not controlling screen sharing - that's the duty of room.ts